docs/usage/getting-started/troubleshooting.md

```diff
@@ -0,0 +1,66 @@
+# Troubleshooting Renovate
+
+Learn how to troubleshoot problems with Renovate, where to find the logging output, and how to get help if needed.
+
+## Getting the logs
+
+Renovate logs the most recent jobs.
+Where you can find the logs depends on how you're running Renovate.
+
+### GitHub Hosted app
+
+Each pull request from Renovate contains a link to the WhiteSource Renovate Dashboard in the PR body text.
+
+The text you're looking for is:
+
+> This PR has been generated by WhiteSource Renovate. View repository job log here.
+
+Click on the blue text "here" to go to the WhiteSource Renovate Dashboard.
```

Comment:
We have a "Dependency Dashboard", which shows pending dependency upgrades.
We also have another dashboard, called "WhiteSource Renovate Dashboard".

This is a bit confusing. Not sure how we can prevent the reader from getting the 2 dashboards mixed up...

Reply:
If all the "WhiteSource Renovate Dashboard" (WSRD) will ever do is give access to the logs, then it would be way clearer if we would call it a "log server" or something similar, as that removes any potential confusion for the end-user.

Reply:
There's always been plans to make it much more, e.g. view extracted dependencies, request new runs, etc. It will be the "real" dashboard eventually, while the issue is more like a small shortcut version.

Reply:
> There's always been plans to make it much more, e.g. view extracted dependencies, request new runs, etc. It will be the "real" dashboard eventually, while the issue is more like a small shortcut version.

Oh cool! So the WSRD will be the full fat "dashboard", and the Dependency Dashboard is the lite version. 😄 

In that case maybe we should keep the terms we have now, as it won't be a log server for ever then... 😄